### PR TITLE
Portfolio contact section 

### PR DESCRIPTION
**Pull request by mwenyoa**
**The following feature where implemented:**
- Add contact section to main page.
- Add background to the contact section.
- Add form inputs, button  and validations.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -205,6 +205,19 @@ <h3>Skills</h3>
...
+        <p class="contact-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="Conact-Form" method="post" action="https://formspree.io/f/mbjwkepg">
+        <input type="text" name="FullName" placeholder="Enter your name" maxlength="30" required>
+         <input type="email" name="email" placeholder="Enter your email" required>
+         <textarea name="message" placeholder="Enter message" maxlength="500" rows="10" required></textarea>
+         <button class="viewpro-btn float-left" type="submit" name="submit-btn">Get in touch</button>
+        </form>
+     </div>
+        <div class="end-line"></div>
+      </section>
```
Comment (prolajumokeoni) [inline index.html]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
sssa  | sty

- [ ] Good job adding all the functionalities,  Kindly change the color of the message text placeholder so your design looks exactly like the Figma design

- [ ] Kindly change the appearance of your button to white it currently has a default background color of `#f7f7f9`

- [ ] Kindly remove the div with the endline as this is not required to be in your project 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi  @mwenyoa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
 - Please fix the horizontal scroll on your website because horizontal scrolling especially for pages with lots of text and buttons to click, it may not be a good experience for users if users have to scroll horizontally to see the remaining content of the page so its best to avoid it in your website, Kindly fix this 👍🏿 


almost-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mwenyoa** [comment]
Hi @prolajumokeoni 
**The following required changes where implemented:**

change2

**Review by keroloussamy — APPROVED** [review]
Hi @mwenyoa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**